app/src/main/java/bankingapp/DBTrans.java

```diff
@@ -176,44 +175,4 @@ public static void deleteAcc(Connection connection, String accountNum) throws SQ
         }
     }
 
-    /**
```

Comment:
It would have been better to pull the UI code out of this method just like you did for the `CreateAccount` class, and to leave the transferFunds method in place in the `DBTrans` class.

After you remove the UI code, the only two things that will remain are:
1. Domain logic (service layer)
2. Database calls (repository layer)

Promoting these two things into the UI layer is just as bad as pushing the UI code down into the repository layer.